scripts/README.md

```diff
@@ -0,0 +1,54 @@
+# Knowledge Base Ingestion
+
+The RAG (Retrieval-Augmented Generation) Knowledge Base Script is designed to automate the process of ingesting documents into a knowledge base, converting them into a vectorized format, and storing this information on the Galadriel chain. This script simplifies the process of creating a decentralized, blockchain-based knowledge base that can be queried and utilized by AI models for enhanced information retrieval and decision-making processes.
+
+## Features
+
+- **Document Ingestion:** Automatically ingest multiple documents from a specified directory.
+- **Vectorization:** Convert textual information into a vector format suitable for AI models.
+- **Blockchain Integration:** Seamlessly integrates with the Galadriel L1 chain, leveraging its Oracle system for decentralized storage and retrieval.
+- **IPFS Support:** Uses IPFS (InterPlanetary File System) for secure and distributed document storage.
+
+## Prerequisites
+
+Before you begin, ensure you have the following:
+- Python 3.11 or later installed on your system.
+- A funded wallet and corresponding private key
+- An API key for `nft.storage` to facilitate document uploading to IPFS. You can obtain this key by registering at [nft.storage](https://nft.storage).
+
+## Setup
+
+To set up your environment for running the RAG Knowledge Base Script, follow these steps:
+
+1. Clone the repository to your local machine.
+2. Create a virtual environment for Python dependencies:
+    ```shell
+    python -m venv venv
+    source venv/bin/activate
+    ```
+3. Install the required Python packages:
+    ```shell
+    pip install -r requirements.txt
+    ```
+4. Create a `.env` file in the root directory of the script and add your `nft.storage` API key and wallet private key as follows:
```

Comment:
root directory of the script can be misunderstood I guess, not sure. Maybe just say add it to `scripts/.env` or somethign along the lines of that?